assets/io.github.CatalystDevOrg.Catalyst.appdata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.CatalystDevOrg.Catalyst</id>
```

Comment:
The ID should be `org.eu.getcatalyst.Catalyst` as the Catalyst website is https://getcatalyst.eu.org

Reply:
Fixed